### PR TITLE
add name: PR (Angel)

### PR DESCRIPTION
**Pull request by angelitora**

### CODE REVIEW

**Review by soerenthomsen — APPROVED** [review]

**Comment by soerenthomsen** [comment]
Let' see who is fastest with providing a review! We always need two official approvals before I can merge :)

**Review by abr099 — APPROVED** [review]

File: training_session_07.md
Comment (hassanimaryam) [inline training_session_07.md:21]: - [Maryam Hassani] (https://github.com/hassanimaryam)

**Review by hassanimaryam — APPROVED** [review]

File: training_session_07.md
Comment (hassanimaryam) [inline training_session_07.md:21]: - [Maryam Hassani](https://github.com/hassanimaryam)

Comment (hassanimaryam) [inline training_session_07.md:21]: - [Maryam Hassani](https://github.com/hassanimaryam)

**Comment by soerenthomsen** [comment]
Thanks @angelitora for this PR and @abr099 and @hassanimaryam for reviewing it :)

I will merge it now!

File: training_session_07.md
Comment (hassanimaryam) [inline training_session_07.md:21]: - [Maryam Hassani](https://github.com/hassanimaryam)

File: training_session_07.md
Comment (hassanimaryam) [inline training_session_07.md:22]: -  [Maryam Hassani](https://github.com/hassanimaryam)

File: training_session_07.md
Comment (hassanimaryam) [inline training_session_07.md:21]: [Maryam Hassani](https://github.com/hassanimaryam)

**Pull request merged**
